NOTICE.txt

```diff
@@ -0,0 +1,9 @@
+                        Base Microservice Example
+                        =========================
+
+Please visit the IDRSolutions implementations of the base microservice for examples of this project in use:
+  * https://github.com/idrsolutions/buildvu-microservice-example
+  * https://github.com/idrsolutions/formvu-microservice-example
+  * https://github.com/idrsolutions/jpedal-microservice-example
+
+Copyright 2018 - 2023 IDRsolutions
```

Comment:
I'm thinking in this one we should have a mention regarding the DefaultFileServlet class which is derived from OmniFaces.

Reply:
I have left the OmniFaces copyright in that class.
Shall we also include it here as follows, I took inspiration from other projects.

Copyright 2018 - 2023 IDRsolutions
This project includes work covered by the following copyright:
DefaultFileServlet.java - Copyright 2021 OmniFaces